packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -61,9 +61,7 @@ export class ForceStartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{
           traceflag.Id,
           traceflag.StartDate,
           traceflag.ExpirationDate,
-          traceflag.DebugLevelId,
-          traceflag.DebugLevel.ApexCode,
-          traceflag.DebugLevel.Visualforce
+          traceflag.DebugLevelId
```

Comment:
No longer need to store previous levels for apex/visualforce since we are deleting the traceflag